### ISSUE TITLE
incomplete unquoted-text definition

### ISSUE THREAD

**Issue by GlenDC** [projectfluent/fluent#18]
Currently the definition of an unquoted-text is as follows:

```EBNF
unquoted-text        ::= ([^{] | '\{')+;
```

source: https://github.com/projectfluent/syntax/blob/master/fluent.ebnf#L28

For most cases this is fine, however when just relying on this definition it breaks in following example:

```FTL
opened-new-window = { brandName[gender] ->
 *[masculine] { brandName } otworzyl nowe okno.
  [feminine] { brandName } otworzyla nowe okno. }
```

In the example given above, it will read the entire line, including the '}' character, which is actually the closing bracket of the root `placeable`. This problem is fixed if you define an `unquoted-text` as follows:

```EBNF
unquoted-text        ::= ([^{}] | '\{' | '\}')+;
```

**Comment by stasm** [comment]
Thanks for the report! `member` is currently defined with the final new line on its end. I've created the 0.3 milestone to focus on the white spacing rules in the syntax. I'll come back to this issue then.

**Comment by stasm** [comment]
This was fixed by #32. We changed the syntax of multiline strings by removing the leading `|` and requiring indentation.
